Mail/Views/Thread List/ThreadListViewModel.swift

```diff
@@ -174,7 +174,7 @@ final class DateSection: Identifiable, Equatable {
     ) {
         assert(folder.isFrozen, "ThreadListViewModel.folder should always be frozen")
         self.mailboxManager = mailboxManager
-        self.folder = folder
+        frozenFolder = folder.freezeIfNeeded()
```

Comment:
We don't need to freezeIfNeeded because of the assert at line 175

Reply:
Yup 👍 
Renaming the contructor.

Reply:
I did that.